src/style.css

```diff
@@ -0,0 +1,159 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  font-family: Avenir, sans-serif;
+}
+
+/* -------big searchBar------- */
+
+.searchBar {
+  margin-top: 20px;
+  height: 70px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.label button {
+  padding: 0;
+  border: none;
+  width: 19px;
+  height: 19px;
+  background: #fff;
+}
+
+.label {
+  display: flex;
+  height: 100%;
+  padding: 0 25px;
+  cursor: text;
+  border-radius: 4px;
+  align-items: center;
+}
+
+.img {
+  width: 19px;
+  height: 19px;
+}
+
+.searchBar input {
+  margin: 0 0 0 14px;
+  background: none;
+  border: none;
+  text-indent: 4px;
+  color: #3d4e61;
+  font-weight: bold;
+  font-size: 16px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.searchBar ::placeholder {
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+  text-shadow: none;
+}
+
+.searchBar:focus-within {
+  background: linear-gradient(#fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2), 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.smSearchBar input {
+  margin-left: 9px;
+  background: none;
+  border: none;
+  color: #3d4e61;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+.searchBar input:focus {
+  line-height: 68px;
+  outline: 0;
+  color: #3d4e61;
+  font-weight: bold;
+  font-size: 16;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+/* -------small searchBar------- */
+
+.smSearchBar {
+  margin-top: 20px;
+  height: 42px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.smLabel button {
+  padding: 0;
+  border: none;
+  width: 11px;
+  height: 11px;
+  background: #fff;
+}
+
+.smLabel {
+  display: flex;
+  height: 100%;
+  padding: 0 13px;
+  cursor: text;
+  border-radius: 4px;
+  align-items: center;
+}
+
+.smImg {
+  display: flex;
+  width: 11px;
+  height: 11px;
+}
+
+.smSearchBar ::placeholder {
+  font-size: 14px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.smSearchBar:focus-within {
+  background: linear-gradient(#fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  box-shadow: none;
+}
+
+.smSearchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.smSearchBar:focus-within:hover {
+  background: linear-gradient(#fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  box-shadow: none;
+}
+
+.smSearchBar input:focus {
+  outline: 0;
```

Comment:
you are duplicating a lot of styles for a small search bar, the correct way to do it: [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 2